kubernetes/admission/test_loadbalancer.rego

```diff
@@ -3,34 +3,47 @@ package library.kubernetes.admission.test_loadbalancer
 import data.library.kubernetes.admission.example_loadbalancer as loadbalancer
 import data.library.kubernetes.admission.input.service
 
+v1beta1 := "admission.k8s.io/v1beta1"
+
+v1 := "admission.k8s.io/v1"
+
 test_clusterip {
-	# i := service.gen_input("ClusterIP", {}, null, null)
+	# i := service.gen_input("ClusterIP", {}, null, null, v1beta1)
```

Comment:
Should we just remove the commented out line here?

Reply:
no opinions. i couldn't figure out the original intent.